### PR TITLE
Add content to page

### PR DESCRIPTION
**Pull request by learnwithalfred**
- Header, hero, main, and footer sections added with content
- Page has been styled using flexbox
- 
Screenshot 2022-04-21 at 16 30 09

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    </div>
+      </a>
+
+    <nav class="">
+      <ul class="nav">
+        <li><a class="nav--item" href="#">Home</a></li>
+        <li><a class="nav--item" href="#">Content</a></li>
+        <li><a class="nav--item" href="#">Call To Action</a></li>
+        <li><a class="nav--item" href="#">Footer</a></li>
+        <li><a class="nav--item" href="#">Sign Up</a></li>
```
Comment (learnwithalfred) [inline index.html:30]: I think you should double-check the BEM naming convention

Screenshot 2022-04-21 at 16 44 42

**Review by learnwithalfred — COMMENTED** [review]
Your work looks great. 

But please check these.
1. Double-check the BEM naming convention.
2.  There is no linter in your application. I recommend you check `super linter` 
3. On Chrome the view looks great 
Screenshot 2022-04-21 at 16 53 11

But in different browsers, it looks different 

Screenshot 2022-04-21 at 16 53 59


Can you look into that?

Thanks

**Review by learnwithalfred — COMMENTED** [review]
Hi @learnwithalfred 

Good job so far!!
There are some issues that you need to work on to finish up the project

Your project has

1. Correct GitHub flow practice

2. Good commit messages
3.  Clean codebase

**Required Changes**
1. I think you should double-check the BEM naming convention
In your code, nav item class is named as `nav--item`
But the BEM naming convention follows the pattern displayed in the image bellow

164510190-4fc61181-0a09-4e54-8c1f-175ae761c206

2. Also can you check the semantic tags again?
 In your code, the logo is not within the nav tag.
The footer too is not using the sematic `footer tag`

**Review by ShahierNashaat — APPROVED** [review]
Hi @learnwithalfred , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**